backend/main.py

```
@@ -870,106 +996,57 @@ async def get_scenario_details(scenario_id: int, db: Session = Depends(get_db)):
-                }
-                for persona in personas
-            ],
-            "scenes": scenes_with_personas
-        }
+            return {"status": "error", "error": "Scenario not found"}
+        return {"status": "ok", "scenario_id": scenario_id, "title": scenario.title, "user": current_user.email}
+    except Exception as e:
+        return {"status": "error", "error": str(e)}
+
+@app.get("/api/scenarios/{scenario_id}/full")
+async def get_scenario_full(scenario_id: int, db: Session = Depends(get_db), current_user: User = Depends(get_current_user)):
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)